### ISSUE TITLE
Unable to setup project for development on Windows.

### ISSUE THREAD

**Issue by ekjotsinghmakhija** [coral-xyz/backpack#3653]
I'm trying to build the project but unable to find the exact **command**.
I have also visited the official documentation.
used the command:
`yarn install`
info Visit [https://yarnpkg.com/en/docs/cli/add](https://yarnpkg.com/en/docs/cli/add) for documentation about this, still nothing seems to work.

<img width="733" alt="Screenshot 2023-04-09 133657" src="https://user-images.githubusercontent.com/92289175/230761826-52341b2e-7145-42f1-ad69-99b94b4244c8.png">

**Comment by richiebijoy** [comment]
Hi @ekjotsinghmakhija 

I am using windows and I am able to build the project. I suggest you visit the link mentioned in the info section of the output. 

A quick google search tells me you need to run this command to upgrade your version of yarn
`npm install --global yarn
`

**Comment by ekjotsinghmakhija** [comment]
Hi @richiebijoy 

I ran the command that you mentioned.
Still it isn't working, just a request from my side can you setup the project in another directory and share the whole process step-by-step.

<img width="960" alt="Screenshot 2023-04-09 203734" src="https://user-images.githubusercontent.com/92289175/230780803-568559c7-3523-4f7f-bd9f-d4c6f4adf552.png">

**Comment by KartikNaik0x** [comment]
hi @richiebijoy when I clone the backpack repository it has all the frontend and backend code do I have to build them seperately and run it?

**Comment by richiebijoy** [comment]
@ekjotsinghmakhija I see you are trying to install on D drive. I think you need to update your env path variables. Check the PATH value in system variable. Let me know what you find. If you find there is no reference to latest version of yarn there then you have to add it there. Should work then.

**Comment by richiebijoy** [comment]
@KartikNaik0x have you forked the repo?? If not do that first. If you are hoping to make contributions to the repo you should fork it and then commit whatever you have done to the forked repo. 

Then you can follow the steps in this video to setup locally, but before this this you will need to do a `yarn install` to get the required node_modules. 

![image](https://user-images.githubusercontent.com/59922104/230783022-2461c3bc-fda8-46cb-bbea-d5f12b2127ab.png)

**Comment by ekjotsinghmakhija** [comment]
@richiebijoy I think I have added the environment variables correctly for yarn at the end.
What is the next step now?

<img width="290" alt="Screenshot 2023-04-09 220530" src="https://user-images.githubusercontent.com/92289175/230784939-99da58e0-39cd-402d-991b-d40f1854a5b6.png">

**Comment by richiebijoy** [comment]
Remove the Path. Try running the install command in powershell. Tell me what the output is/

**Comment by KartikNaik0x** [comment]
@richiebijoy yes I forked the repo ran yarn install and got the node_modules . I built the project and followed that video but I get this error under the extension when I drag drop the build folder in the extension section of the browser.
[Screenshort](https://imgur.com/a/wMK3fHd)

**Comment by richiebijoy** [comment]
You have to copy the dev folder for development there. Not the build folder.

**Comment by KartikNaik0x** [comment]
@richiebijoy  Yeah got it . Thanks

**Comment by sp-sandeshpokhrel** [comment]
@ekjotsinghmakhija I think your yarn installation didn't go correctly, you should use the following command for 3.0.0 version install.
```shell
npm install --global yarn
```

**Comment by fancymithun** [comment]
I am using windows system I was able to install dependencies but I was issue with yaen build.

Error message : D:\xNFT\backpack\packages\common) yarn run build exited (127)

**Comment by ekjotsinghmakhija** [comment]
@fancymithun Yes I was able make it work, you need to install yarn version correctly, the required one which can be found on yarn's official website.

**Comment by fancymithun** [comment]
@ekjotsinghmakhija  I am using yarn version 3.5.0.

what version of yarn need to used in this case?

**Comment by ekjotsinghmakhija** [comment]
@fancymithun you don't have to do anything you already have the required version yarn ^3.5.0 .
